R/add_transition_risk_category_at_company_level.R

```diff
@@ -0,0 +1,14 @@
+add_transition_risk_category_at_company_level <- function(data) {
```

Comment:
If we  have `add_transition_risk_category_at_company_level(), then do we need to rename `add_transition_risk_category()` to `add_transition_risk_category_at_product_level()`?

Reply:
Right! I created a separate issue for this work: https://github.com/2DegreesInvesting/tiltTransitionRisk/issues/22